.github/workflows/bot-ci-failure.yml

```
@@ -0,0 +1,89 @@
+          PR_NUMBER=$(gh pr list --repo "$REPO" --state open --limit 100 --json number,headRefOid --jq ".[] | select(.headRefOid == \"$HEAD_SHA\") | .number" | head -n 1)
+          if [ -n "$PR_NUMBER" ]; then
+             echo "Found PR #$PR_NUMBER by scanning open PRs."
+             emit_pr "$PR_NUMBER"
+             exit 0
+          fi
+          echo "::warning::No open PR found. This workflow run might not be attached to an open PR."
+          exit 0
+
+  call-ci-failure-bot:
+    needs: find-pr
+    timeout-minutes: 15
```

actionlint (1.7.12)

[error] when a reusable workflow is called with "uses", "timeout-minutes" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "call-ci-failure-bot"

(syntax-check)